packages/pn-pa-webapp/src/components/NewNotification/Attachments.tsx

```diff
@@ -49,14 +56,18 @@ const AttachmentBox: React.FC<AttachmentBoxProps> = ({
   fieldLabel,
   fieldValue,
   fieldTouched,
-  fieldErros,
+  fieldErrors,
   onFieldTouched,
   onFileUploaded,
   onRemoveFile,
   fileUploaded,
 }) => {
+  const [focused, setFocused] = useState(false);
```

Comment:
Why we need this?

Reply:
We need to show helper text of max characters always when input is focused

Reply:
Ok we can avoid the use of useState to prevent unnecessary re-rendering. Try to use the [useFormControl](https://mui.com/material-ui/react-text-field/#useformcontrol)